draft-ietf-ppm-dap.md

```diff
@@ -1867,6 +1874,10 @@ following checks:
    skew. If a report is rejected for this reason, the Aggregator SHOULD mark the
    input share as invalid with the error `report_too_early`.
 
+1. Check if the report's timestamp is before the task's `task_start` time. If
```

Comment:
I think this makes it slightly more clear that you don't mean "smaller than or equal to".
```suggestion
1. Check if the report's timestamp is smaller than the task's `task_start` time. If
```

Reply:
I changed it to `earlier than`.